Cargo.toml

```diff
@@ -19,6 +19,9 @@ piet-common = "0.0.2"
 piet = "0.0.2"
 kurbo = "0.2.1"
 
+[[bin]]
```

Comment:
I don't think this is necessary, you run the example with `cargo run --example calc`. It is very much an example, not really a useful binary in and of itself.

Reply:
Heh, I wasn't quite sure, so I just added this so I could test it